<def f='llvm/llvm/include/llvm/Support/Error.h' l='176' ll='179' type='void llvm::Error::Error()'/>
<use f='llvm/llvm/include/llvm/Support/Error.h' l='889' u='c' c='_ZN4llvm12handleErrorsENS_5ErrorEDpOT_'/>
<doc f='llvm/llvm/include/llvm/Support/Error.h' l='175'>/// Create a success value. Prefer using &apos;Error::success()&apos; for readability</doc>
